src/endpoints/v1.es6.js

```diff
@@ -144,9 +155,20 @@ class APIv1Endpoint {
     });
   }
 
+  log (state, method, uri, options, status, err, duration) {
+    if (this.config.debugLevel === 'info') {
```

Comment:
Yeah... I suppose I should use an enum instead.
On Nov 4, 2015 1:32 PM, "Chris Dary" <notifications@github.com> wrote:

> In src/endpoints/v1.es6.js
> <https://github.com/reddit/snoode/pull/59#discussion_r43942658>:
>
> > @@ -144,9 +155,20 @@ class APIv1Endpoint {
> >      });
> >    }
> >
> > +  log (state, method, uri, options, status, err, duration) {
> > +    if (this.config.debugLevel === 'info') {
>
> I guess we don't have integer equivalents for >= comparisons of debug
> level? Do we have a "debug" level for debugLevel? If so, shouldn't this
> also still run?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reddit/snoode/pull/59/files#r43942658>.
>